.gitlab-ci.yml

```diff
@@ -156,6 +156,23 @@ test-srml-staking:                 &test-srml-staking
     - time cargo test --release --verbose --no-default-features --features std
     - sccache -s
 
+test-full-crypto-feature:          &test-full-crypto-feature
+  stage:                           test
+  <<:                              *docker-env
+  variables:
+    # Enable debug assertions since we are running optimized builds for testing
+    # but still want to have debug assertions.
+    RUSTFLAGS: -Cdebug-assertions=y
+    RUST_BACKTRACE: 1
+  except:
+    variables:
+      - $DEPLOY_TAG
+  script:
+    - cd core/primitives/
+    - time cargo build --release --verbose --no-default-features --features full_crypto
```

Comment:
Please don't build it in release, that just waste resources^^

Reply:
ok.